packages/main/src/themes/base/SliderBase-parameters.css

```diff
@@ -9,14 +9,14 @@
 	--_ui5_slider_handle_width: 1.625rem;
 	--_ui5_slider_handle_border: solid 0.125rem var(--sapField_BorderColor);
 	--_ui5_slider_handle_background: var(--sapButton_Background);
-	--_ui5_range_slider_handle_background: rgba(var(--sapButton_Background), 0.25);
+	--_ui5_range_slider_handle_background: rgba(255, 255, 255, 0.25);

```

Comment:
On hover over the progress bar in Internet Explorer, the handles are still not transparent.

Reply:
My mistake, it works as expected :)